releases/net472/KnownIssues/613745 - Single-name references are removed by the SDK when targeting 4.7.2.md

```diff
@@ -0,0 +1,23 @@
+# Single-name references are removed by the SDK when targeting 4.7.2 and referencing a package with the same assembly.
+
+## Symptoms
+When a project targets 4.7.2 and references an assembly with both a single name reference and a package reference (either a explicit or a transitive package reference), the assembly will not be passed as a reference to the compiler, causing build time errors for types that are not found. An example of a build error caused by this issue is:
+```
+error CS0246: The type or namespace name 'HttpClient' could not be found (are you missing a using directive or an assembly reference?)
+```
+
+Candidate assemblies that might run into this issue are: `System.Diagnostics.Tracing.dll`, `System.IO.Compression.dll`, and `System.Net.Http.dll`. Also is worth noting that this issue will only happen when the project uses `<PackageReference` model instead of `packages.config` to manage their NuGet dependencies.
```

Comment:
- delete the colon.
- Also is worth noting that this issue --> This issue
- will only happen --> only happens
- PackageReference model --> the PackageReference model
their --> its